doc/release-notes.md

```diff
@@ -32,6 +32,100 @@ PIVX Core should also work on most other Unix-like systems but is not frequently
 Notable Changes
 ==============
 
+## zPIV Public Spends
+
+Recent exploits of the Zerocoin protocol (Wrapped serials and broken P1 proof) required us to enable the zerocoin spork and deactivate zPIV functionality in order to secure the supply until the pertinent review process was completed.
+
+Moving forward from this undesired situation, we are enabling a secure and chain storage friendly solution for the zerocoin public spend (aka zPIV to PIV conversion).
+
+The explanation of how this works can be found in #891
+
+After block `1,880,000` has past, `SPORK_16` will be deactivated to allow zPIV spends to occur using this new public spend method. The minting of **NEW** zPIV, as well as zPIV staking will remain disabled for the time being.
```

Comment:
Release notes should probably mention that the "public zerocoin spend" feature:
- is available only for v2 coins thus v1 coins will be still unspendable after block 1,880,000.
- removes the zerocoin privacy forcing the spender to reveal the mint he's about to spend.